services/todo-legacy/repositories/todo.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  AnyObject,
+  DataSourceConstructor,
+  DefaultCrudRepository,
+  Filter,
+  Model,
+  ModelBaseConstructor,
+  ObjectType,
+  Repository,
+  Where,
+  juggler
+} from '@loopback/repository';
+import { inject, Application } from '@loopback/core';
+
+export class TodoRepository {
+  // FIXME(kev): The juggler.ModelBase type doesn't carry any definitions
+  // for the CRUD functions that typically get mixed in when createModel
+  // makes a definition, so we have to use type "any" to avoid complaint!
+  model: juggler.ModelBase | any;
```

Comment:
I find this rather ugly. Can `juggler` export an interface for CRUD operations, so that `model` can be typed as `juggler.ModelBase | juggler.CrudApi`? Or perhaps `juggler.CrudModel`?

@raymondfeng